src/wx/panel.cpp

```diff
@@ -2135,6 +2145,42 @@ GLDrawingPanel::GLDrawingPanel(wxWindow* parent, int _width, int _height)
     if (!did_init) DrawingPanelInit();
 }
 
+void GLDrawingPanel::ChangeVsync(bool enable)
+{
+    int vsync = enable ? 1 : 0;
+// non-portable vsync code
+#if defined(__WXGTK__) && defined(GLX_SGI_swap_control)
+    static PFNGLXSWAPINTERVALSGIPROC si = NULL;
+
+    if (!si)
+        si = reinterpret_cast<PFNGLXSWAPINTERVALSGIPROC>(glXGetProcAddress(reinterpret_cast<const unsigned char*>("glXSwapIntervalSGI")));
```

Comment:
@rkitover This right here is a very big YIKES. This was legit broken previously... I have no idea how we did not get a segfault at some point.

Reply:
Don't worry, we got all kinds of segfaults at all points. Like 99% of the github issues is some kind of crash.